### DISCUSSION TITLE
How to change the background color of the fancy tab bar?

### DISCUSSION THREAD

**Discussion by santigo-dev** [wezterm/wezterm#1703] [Q&A]
Hello! Well basically I was changing the colorscheme of wezterm, and I realized that I never changed the old colorscheme's tab bar, and was using the default colors since I don't use tabs that much.

As of right now I'm using the fancy tab bar, when I was changing the colors I read the comment left on the wiki

```lua
-- The color of the strip that goes along the top of the window
-- (does not apply when fancy tab bar is in use)
```

So is the setting in another place?

**Comment by wez** [comment]
https://wezfurlong.org/wezterm/config/appearance.html#native-fancy-tab-bar-appearance has information on styling the fancy tab bar
